docs/reference/upgrade-to-v2.md

```diff
@@ -1,7 +1,17 @@
 ---
 mapped_pages:
   - https://www.elastic.co/guide/en/apm/agent/nodejs/current/upgrade-to-v2.html
----
+applies_to:
+  stack:
+  serverless:
+    observability:
+  product:
+    apm_agent_node: ga
+products:
+  - id: cloud-serverless
+  - id: observability
+  - id: apm
+   ---
```

Comment:
This is causing [rendering issues](https://docs-v3-preview.elastic.dev/elastic/apm-agent-nodejs/pull/4739/reference/upgrade-to-v2).

```suggestion
---
```